examples/projections/table/README.txt

```diff
@@ -49,7 +49,7 @@ The below table shows the projection codes for the 31 GMT projections.
      - :doc:`Eckert VI equal area </projections/misc/misc_eckertVI>`
    * - **L**\ |lon0|/|lat0|/|lat1|/|lat2|/\ *width*
      - :doc:`Lambert conic conformal </projections/conic/conic_lambert>`
-   * - **M**\ [|lon0|\ [/|lat0|]/]\ *width*
+   * - **M**\ |lon0|\ [/|lat0|]/]\ *width*
```

Comment:
For the Mercator (cylindrical) projection the PyGMT table entry is correct, but the documentation has to be updated (please see https://www.pygmt.org/dev/projections/cyl/cyl_mercator.html).
```suggestion
   * - **M**\ [|lon0|\ [/|lat0|]/]\ *width*
```

Reply:
Thanks for the correction @willschlitzer!
Do you plan to update the Mercator (cylindrical) projection documentation in a separate PR? Please see also my comment below regarding other projections.